backend.py

```diff
@@ -0,0 +1,30 @@
+# Class/Backend
+
+class Backend:
+    def construct_query(self, search_parameters: list[tuple]) -> str:
+        pass
+            # Constructs a query string from the given search parameters
+            # Arguments: search_parameters (list[tuple]): A list of tuples representing the searc h parameters
+            
+            # Returns: str: The constructed query string
+
+
+    def query_api(self, query: str)->list['Card']:
```

Comment:
```def query_api(self, query: str)->list['Card']:```

- [ ] **Card**: is a type, remove the apostrophes. 
- [ ] **Spacing**: Put a space on both sides of `->`